🤖 Auto-merge documentation from branches

buger opened the pull request:
### **User description**
## 📚 Documentation Merge Summary

This PR contains automatically merged documentation from multiple branches.

**Triggered by:**
- **Branch:** N/A
- **Commit:** N/A
- **PR:** N/A (direct push)

**Deployment Details:**
- **Generated from:** `branches-config.json`  
- **Run ID:** 19560465986
- **Subfolder:** `(root deployment)`
- **Timestamp:** $(date)

### Changes Include:
- ✅ Merged documentation from multiple branches
- ✅ Generated unified `docs.json` configuration
- ✅ Updated assets and content structure
- ✅ Cleaned up outdated version folders

---

🚦 **This PR will be processed by merge queue to ensure proper validation and ordering.**

Previous deployment PRs have been automatically closed to prevent conflicts.


___

### **PR Type**
Documentation


___

### **Description**
- Add 5.8.8 Dashboard release notes

- Add 5.8.8 Gateway release notes

- Add 2.8.6 MDCB release notes

- Update LTS/latest versions in overviews


___

### Diagram Walkthrough


```mermaid
flowchart LR
  overview["Release overview metadata"] -- "update LTS/latest" --> dash["Dashboard 5.8.8 notes"]
  overview -- "update LTS/latest" --> gw["Gateway 5.8.8 notes"]
  overview -- "update latest" --> mdcb["MDCB 2.8.6 notes"]
```



<details> <summary><h3> File Walkthrough</h3></summary>

<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><details><summary>8 files</summary><table>
<tr>
  <td><strong>dashboard.mdx</strong><dd><code>Add Dashboard 5.8.8 notes with CVE fixes, Go env</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1002/files#diff-46eab3c33b18d7ca160d2569a5d0575603ce1af6ba1af53ae2a943aff9926bb4">+93/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td><strong>gateway.mdx</strong><dd><code>Add Gateway 5.8.8 notes and downloads/CVEs</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1002/files#diff-9095026f67d8773f400fd494ced0d00ea8034dc1ccd14aa0988038ca8f45d0be">+81/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td><strong>mdcb.mdx</strong><dd><code>Add MDCB 2.8.6 notes with security fixes</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1002/files#diff-55c4580825f8a4e21f843de53a5d785dc9effd398ba7699192ee8d628eb05153">+63/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td><strong>overview.mdx</strong><dd><code>Update LTS and latest versions for LTS streams</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1002/files#diff-dd436baaf553c4f68b417769481419779095437ed991a35723c1067a6af182a6">+6/-6</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td><strong>dashboard.mdx</strong><dd><code>Add nightly Dashboard 5.8.8 notes mirrored</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1002/files#diff-e85921379c77a82273626b506f59a89e409d94c7eef418e457a470ef0cf04248">+93/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td><strong>gateway.mdx</strong><dd><code>Add nightly Gateway 5.8.8 notes mirrored</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1002/files#diff-66e7dd330395b86c4d022a736179f57d8d49fb3073fb83d4d3228ee1055470c7">+81/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td><strong>mdcb.mdx</strong><dd><code>Add nightly MDCB 2.8.6 notes mirrored</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1002/files#diff-a0cdf629a6faebbbc05dd4d608992df3ef875fdd08e1a1f2162549d8b1677232">+63/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td><strong>overview.mdx</strong><dd><code>Update nightly overview LTS/latest metadata</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1002/files#diff-fb8ca7bde79e266244d0bcf78f942b283149eefc86d9437a7ea05f28c48c930d">+6/-6</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></details></td></tr></tr></tbody></table>

</details>

___

marksou commented:
Hi @adelowo, the `config` section is now `tyk-configuration-reference`. Can you resolve the merge conflict for the 2 pages?

adelowo commented:
Sure. Will do that

adelowo commented:
Will sort merge conflicts tomorrow @marksou

buger commented:
@adelowo we need to add docs on at which part of Tyk KV store can be used

adelowo commented:
@buger PTAL https://github.com/TykTechnologies/tyk-docs/pull/1002/files#diff-6f0f408379b71ecc0cdd2b775af21c41R20-R25

@marksou fixed merge conflicts

marksou commented:
Merge as part of 2.10

buger committed:
🤖 Auto-merge documentation from branches

Generated from branches-config.json at

- Merged documentation from multiple branches
- Generated unified docs.json
- Updated assets and content structure
